aten/src/ATen/test/basic.cpp

```diff
@@ -138,7 +138,8 @@ void TestLoadOfAddsWithCopy(DeprecatedTypeProperties& type) {
                    end - begin)
                    .count()
             << " ms" << std::endl;
-  ASSERT_EQ_RESOLVED(norm(100000 * d).item<double>(), norm(r).item<double>());
+  
+  ASSERT_EQ_RESOLVED(norm(1000 * d).item<double>(), norm(r).item<double>());
```

Comment:
Why changed to a smalled size?  see 
```
>>> a=torch.ones(3, 4).half().cuda()
>>> b=a*100000
>>> b
tensor([[inf, inf, inf, inf],
        [inf, inf, inf, inf],
        [inf, inf, inf, inf]],  dtype=torch.float16)
>>> r = torch.zeros(3, 4).half()
>>> for i in range(100000):
...     r = torch.add(r, a)
...
>>> r
tensor([[2048., 2048., 2048., 2048.],
        [2048., 2048., 2048., 2048.],
        [2048., 2048., 2048., 2048.]], dtype=torch.float16)
```
If we use float as accumulate type, the `r` input can get a `7096 number`, but the `b` input gets an `inf` number. I checked cuda path, this case is also failed for such a big input.